core/templates/dev/head/components/score-ring/score-ring.directive.ts

```diff
@@ -49,6 +50,26 @@ oppia.directive('scoreRing', [
               setScore(newScore);
             }
           });
+
+          ctrl.getScoreRingColor = function() {
+            if (ctrl.isTestPassed()) {
+              // return color green when passed.
+              return 'rgb(0, 150, 136)';
```

Comment:
Don't repeat the same rgb strings. Use constants.

Reply:
Done.